.ci/scripts/dart_unit_tests_win32.sh

```diff
@@ -3,5 +3,5 @@
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
 
-dart pub global run flutter_plugin_tools test --packages-for-branch \
-  --log-timing
+dart pub global run flutter_plugin_tools test --exclude=../../script/configs/windows_unit_tests_exceptions.yaml \
```

Comment:
The LUCI recipe runs scripts with the repo root as the working directory, so this should be `--exclude=script/configs/...`